src/utils.tsx

```diff
@@ -23,6 +23,15 @@ type User = {
   attributes: optimizely.UserAttributes;
 };
 
+export type UserInfo = {
+  id: string;
+  attributes?: optimizely.UserAttributes;
+};
```

Comment:
We already have almost the same in `client.ts`: `UserContext`. Why not use that instead of creating this?

Reply:
Its the same one. so instead of importing `UserInfo` from client.ts into utils, it made more sense to me to import something from utils in to Client. So i moved the same `UserInfo` type that you are talking about from client to utils and used it in both the places.